README.md

```diff
@@ -1,2 +1,175 @@
-# air-pollution-aqi
-This project is about building a mobile web application to check a list of air pollution metrics about dynamic changes in different places that I have created using of React and Redux.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>AQI-Weather Metric Mobile App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 AQI-Weather Metric Mobile App<a name="about-project"></a>
+
+**AQI-Weather Metric Mobile App** This project is about building a mobile web application to check a list of air pollution metrics about dynamic changes in different places that I have created using React and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML-5
+- CSS3
+- Reactjs
+- Redux
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://www.w3schools.com/REACT/DEFAULT.ASP">Reactjs</a></li>
+    <li><a href="https://www.tutorialspoint.com/redux/index.htm">Redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add the real data views for cities in relation to air pollution**
+- **Search and filter cities**
+- **View population of people in a city**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://climaticvals.onrender.com/)
```

Comment:
* The live demo link doesn't work as expected, kindly ensure the home and details page works